Core/clim-core/commands/commands.lisp

```diff
@@ -101,7 +103,8 @@
      ;; sub-menus.
      (check-type value (or command-table symbol cons))
      (when (listp value)
-       (setf value (menu-items-from-list value))))
+       (let ((items (menu-items-from-list value)))
+         (setf value (make-command-table nil :menu items)))))
```

Comment:
Please exercise this in one of the examples/tests.

Reply:
fixed (extended menu-test)